torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -1402,7 +1402,9 @@
     long s1 = THTensor_(size)(LIBRARY_STATE ((THPTensor*)$arg4)->cdata, 0);
     long s2 = THTensor_(size)(LIBRARY_STATE ((THPTensor*)$arg5)->cdata, 0);
     THTensor_(resize2d)(LIBRARY_STATE ((THPTensor*)$arg0)->cdata, s1, s2);
+    #ifndef WITH_CUDA
```

Comment:
#if !IS_CUDA didn't build, was trying to grab the cuda tensor specific compiler def.